### DISCUSSION TITLE
What is the scheme and host for in GoTrueConfig?

### DISCUSSION THREAD

**Discussion by unovil** [supabase-community/supabase-kt#263] [Q&A] [answered]
While adapting the [android-login](../tree/master/demos/android-login) demo for my project, I found this code (adapted to my project):
```kt
fun GoTrueConfig.platformGoTrueConfig() {
    scheme = "io.jan.supabase"
    host = "login"
}
```

Which was then used in
```kt
install(GoTrue) {
    platformGoTrueConfig()
}
```

I couldn't find documentation about `scheme` and `host`, other than that they are used for deep linking, or what I should change these string values to, if needed. I'm only using the OneTap signin option from Google, so I don't know if these are required.

**Comment by jan-tennert** [comment, marked as answer]
sorry if thats a bit late, I dont get notifications for discussions, the host and scheme are for deeplink callbacks when using OAuth. If you are using onetap, these are not required
